### PR TITLE
Fix loader relationship failures

### PR DESCRIPTION
**Pull request by theofidry**
Closes #206

### CODE REVIEW

**Comment by egeloen** [comment]
@theofidry Thank you for this fix :)

**Comment by geoffrey-brier** [comment]
@theofidry Awesome work man :+1:

File: src/Alice/DataFixtures/Loader.php [hunk 1]
```diff
@@ -198,13 +197,13 @@ private function tryToLoadFiles(PersisterInterface $persister, array &$normalize
...
 
-                $objects = array_merge($objects, $dataSet);
+                $references = array_merge($references, $dataSet);
             } catch (\UnexpectedValueException $exception) {
                 $this->registerErrorMessage($fixtureFilePath, $exception->getMessage());
             }
         }
 
-        return $objects;
+        return $references;
```
Comment (egeloen) [inline src/Alice/DataFixtures/Loader.php]: If you directly returns the references, you can remove the array merge done line 90 :)

Comment (theofidry) [inline src/Alice/DataFixtures/Loader.php]: good shout, and as `array_merge` is slow, that's another big win

**Comment by theofidry** [comment]
Here you go, a bit sad to do two releases the same day but quite happy to have solved that problem 😄

**Pull request merged**